test-app/config/ember-try.js

```diff
@@ -11,6 +11,11 @@ module.exports = async function () {
         name: 'ember-lts-3.24',
         npm: {
           devDependencies: {
+            '@types/ember-qunit': '^5.0.0',
+            '@types/ember-resolver': '^5.0.11',
+            '@types/ember__test-helpers': '^2.6.1',
+            'ember-qunit': '^5.1.5',
+            'ember-resolver': '^8.0.3',
```

Comment:
Note that these are *not* the latest versions—I'm good with the bump nonetheless, but we'll want to bump *further* as well.

Reply:
Oh, I see – this is for the `ember-try` config for the old LTS. Reading fail!